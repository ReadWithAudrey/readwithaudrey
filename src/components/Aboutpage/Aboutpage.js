import React from 'react'
import AboutpageStyled from './Aboutpage.style'
import PropTypes from 'prop-types'

const Aboutpage = () => (
  <AboutpageStyled>
    <div id="what-is-audrey">
      <h2 className="f2 pink tc">What is Audrey?</h2>
      <p className="tc lh-copy">Audrey is a platform which matches readers with listeners to read thought-provoking fiction and non-fiction aloud together.</p>
    </div>
    <div id="who-for">
      <h2 className="f2 pink tc">Who is it for?</h2>
      <p className="tc lh-copy">People who enjoy reading and social connection, and like to hear diverse viewpoints.</p>
    </div>
    <div id="who-with">
      <h2 className="f2 pink tc">Who do I read with?</h2>
      <p className="tc lh-copy">We match people with shared interests, but diverse life journeys. You can choose to be either the reader or listener.</p>
    </div>
  </AboutpageStyled>
)

Aboutpage.propTypes = {
  siteTitle: PropTypes.string,
}

export default Aboutpage