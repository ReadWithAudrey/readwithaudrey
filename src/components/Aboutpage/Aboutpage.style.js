import styled from "styled-components"

const AboutpageStyled = styled.div.attrs({
  className: "bg-light-green flex flex-column items-center justify-between ph4 c-padding montserrat w-100",
})`
  height: 100vh;
`
export default AboutpageStyled;