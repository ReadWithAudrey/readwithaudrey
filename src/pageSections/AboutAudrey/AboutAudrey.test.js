import React from 'react'
import renderer from 'react-test-renderer'
import Aboutpage from './AboutAudrey'

describe('Aboutpage component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<Aboutpage />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
