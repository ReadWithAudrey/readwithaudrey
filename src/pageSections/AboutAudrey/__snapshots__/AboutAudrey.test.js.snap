// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aboutpage component snapshot test renders correctly 1`] = `
<div
  className="sc-bdVaJa bg-light-green flex flex-column items-center justify-between pt4 pb5 ph4 montserrat w-100 fePDys"
>
  <div
    id="what-is-audrey"
  >
    <h2
      className="f2 pink tc"
    >
      What is Audrey?
    </h2>
    <p
      className="sc-jzJRlG montserrat f5 tc lh-copy etWKQu"
    >
      Audrey matches readers with listeners to read thought-provoking books aloud together, sparking interesting conversations and creating meaningful relationships.
    </p>
  </div>
  <div
    id="who-for"
  >
    <h2
      className="f2 pink tc"
    >
      Who is it for?
    </h2>
    <p
      className="sc-jzJRlG montserrat f5 tc lh-copy etWKQu"
    >
      Anyone who enjoys reading, would like to develop a sense of connectedness with someone new, and values other people’s perspectives.
    </p>
  </div>
  <div
    id="who-with"
  >
    <h2
      className="f2 pink tc"
    >
      Who do I read with?
    </h2>
    <p
      className="sc-jzJRlG montserrat f5 tc lh-copy etWKQu"
    >
      From within our community, we’ll take care to identify a reading partner we think you’ll connect well with. Someone with some shared interests to you.
    </p>
  </div>
</div>
`;
